src/libraries/pkg/Microsoft.NETCore.Platforms/runtimeGroups.props

```diff
@@ -130,6 +130,34 @@
       <Versions>11;12;13</Versions>
     </RuntimeGroup>
 
+    <RuntimeGroup Include="solaris">
```

Comment:
Do all of these match what is returned by `uname`?

Reply:
`uname -o` returns `illumos` on illumos based distros. On Solaris, `-o` is not a switch so we distinguish them by that in shell script. In C/C++ and C# code, we use `struct utsname` (which uname utility also internally uses for some options) to get elaborate classification.

Reply:
I see the special casing...  let me comment on that.